drivers/watchdog/wdt_handlers.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <syscall_handler.h>
+#include <watchdog.h>
```

Comment:
Note that this can allow protected processes to reboot the system.  Not sure that's really in-scope for "protection" or not.

Basically this is another spot where I don't really see a useful real-world case where you'd want to use this outside the kernel.

Reply:
i thought it may be useful to at least allow user threads to kick the dog.
but yeah I can't think of a concrete use-case, will drop this.